pcsx2/gui/Dialogs/AboutBoxDialog.cpp

```diff
@@ -37,69 +37,53 @@ Dialogs::AboutBoxDialog::AboutBoxDialog(wxWindow* parent)
 		wxDefaultPosition, wxDefaultSize
 		)
 {
-	// [TODO] : About box should be upgraded to use scrollable read-only text boxes.
-
-	wxString developsString = wxsFormat(
-		L"avih, Refraction, rama, pseudonym, gregory.hainaut, turtleli"
-		L"\n\n"
-		L"%s: \n"
-		L"Arcum42, Aumatt, drk||raziel, "
-		L"cottonvibes, gigaherz, saqib, "
-		L"\n"
-		L"Alexey Silinov, Florin, "
-		L"goldfinger, Linuzappz, loser, "
-		L"Nachbrenner, shadow, Zerofrog, tmkk, Jake.Stine"
-		L"\n\n"
-		L"%s:\n"
-		L"Falcon4ever, Bositman, Akasha",
-		_("Previous versions"), _("Webmasters"));
-
-	wxString contribsString = wxsFormat(
-		L"ssakash, FlatOut, CK1, lightningterror, atomic83github, Pandubz, Prafull, Blyss Sarania, Nobbs66, Alessandro Vetere"
-		L"\n\n"
-		L"%s: \n"
-		L"ChickenLiver (Lilypad), Gabest (GSdx, Cdvdolio, Xpad), Ckemu, "
-		L"General Plot, KrossX, Devina, micove, black_wd, Belmont, "
-		L"\n"
-		L"BGome, _Demo_, Dreamtime, Hiryu and Sjeep, nneeve, F|RES, "
-		L"Shadow Lady, MrBrown, razorblade, Seta-san, Skarmeth, feal87, Athos"
-		L"\n",
-		_("Previous versions"));
-
-	wxFlexGridSizer& boxesContainer = *new wxFlexGridSizer(2, 0, StdPadding);
-	boxesContainer.AddGrowableCol(0, 1);
-	boxesContainer.AddGrowableCol(1, 1);
-
-	wxStaticBoxSizer& developsBox = *new wxStaticBoxSizer(wxVERTICAL, this);
-	wxStaticBoxSizer& contribsBox = *new wxStaticBoxSizer(wxVERTICAL, this);
-
-	pxStaticText& developsText = Text(developsString).SetMinWidth(350);
-	pxStaticText& contribsText = Text(contribsString).SetMinWidth(350);
-
-	developsBox += Heading(_("Developers")).Bold() | StdExpand();
-	developsBox += developsText | StdExpand();
-	contribsBox += Heading(_("Contributors")).Bold() | StdExpand();
-	contribsBox += contribsText | StdExpand();
-
-	boxesContainer += developsBox | StdExpand();
-	boxesContainer += contribsBox | StdExpand();
-
 	// Main layout
+
+	SetMinWidth(450);
+
 	*this += m_bitmap_logo | StdCenter();
 
 	*this += Text(_("PlayStation 2 Emulator"));
 
 	*this += new wxHyperlinkCtrl(this, wxID_ANY,
-		_("PCSX2 Official Website and Forums"), L"https://pcsx2.net"
+		_("Main Website"), L"https://pcsx2.net"
 		) | pxProportion(1).Center().Border(wxALL, 3);
 
 	*this += new wxHyperlinkCtrl(this, wxID_ANY,
-		_("PCSX2 Official Git Repository at GitHub"), L"https://github.com/PCSX2/pcsx2"
+		_("Support Forums"), L"https://forums.pcsx2.net"
+	) | pxProportion(1).Center().Border(wxALL, 3);
+
+	*this += new wxHyperlinkCtrl(this, wxID_ANY,
+		_("Git Repository at GitHub"), L"https://github.com/PCSX2/pcsx2"
 		) | pxProportion(1).Center().Border(wxALL, 3);
 
-	*this += boxesContainer | StdCenter();
+	*this += new wxHyperlinkCtrl(this, wxID_ANY,
+		_("Contributors"), L"https://github.com/PCSX2/pcsx2/graphs/contributors"
```

Comment:
This ignores the old Google code contributors though.

Reply:
How about something like this?
![image](https://user-images.githubusercontent.com/18107717/65576752-d5ef3400-df72-11e9-8758-5f4e5ecc022e.png)

Link1 links to github, link2 to https://pcsx2.net/meet-the-team.html

Reply:
Looks really weird to be honest, why not just have a singular link to OpenHub instead which saves all contributor details?

Reply:
I'm not a fan of "Link1" and "Link2" either. Doesn't look very clean :P

Reply:
We can have all this information on the website, contributors/developers haven't been updated since 1.2.1 release, maybe @bositman can cook something up that provides info for 1.2.1 and below releases (google code) and 1.4 and above github.
Feel free to leave suggestions.

Reply:
This it?
https://www.openhub.net/p/Pcsx2/contributors/summary

Reply:
Nice, this lists everyone.